wwwbase/search.php

```diff
@@ -87,6 +94,14 @@
   $isAllDigits = $arr[2];
 }
 
+if(SPOOF_ENABLED && $cuv) {
+  $cuv_normalized = SPOOF_NORMALIZE ? mb_strtolower(StringUtil::unicodeToLatin($cuv)) : $cuv;
+  $cuv_spoofed = SPOOF_WORDS[$cuv_normalized];
```

Comment:
Please add a `@` here (as in `@SPOOF_WORDS[...`). This suppresses a warning in case no spoof words are defined (i.e., the `spoofWords` section is commented out in `dex.conf`).